flyway/sql/V1.0.0__init.sql

```diff
@@ -1,13 +1,37 @@
 -- テーブル作成
 -- ユーザマスタ
-CREATE TABLE m_user
+CREATE TABLE users
 (
-    id                        int unsigned NOT NULL AUTO_INCREMENT COMMENT 'ID',
-    created_at                datetime            DEFAULT CURRENT_TIMESTAMP COMMENT '登録日時',
-    updated_at                datetime            DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT '更新日時',
-    deleted_at                datetime            DEFAULT NULL COMMENT '削除日時',
-    PRIMARY KEY (id)
+    user_id  int unsigned NOT NULL AUTO_INCREMENT COMMENT 'user_id',
+    email    varchar(100) NOT NULL COMMENT 'email',
+    password varchar(100) NOT NULL COMMENT 'password',
+    PRIMARY KEY (user_id)
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8mb4
   COLLATE = utf8mb4_unicode_ci comment ='ユーザマスタ'
-;
\ No newline at end of file
+;
+
+-- 投稿マスタ
+CREATE TABLE posts
+(
+    post_id int unsigned NOT NULL AUTO_INCREMENT COMMENT 'post_id',
+    user_id int unsigned NOT NULL COMMENT 'user_id',
+    body    text NOT NULL COMMENT 'body',
+    date    datetime DEFAULT CURRENT_TIMESTAMP COMMENT 'date',
+    PRIMARY KEY (post_id)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8mb4
+  COLLATE = utf8mb4_unicode_ci comment ='投稿マスタ'
```

Comment:
[nits]
`=` の右側の半角スペースが欲しいのと、
折角インデントが揃えてあるので、COMMENTも改行した方が見栄えが良いと思いました。

```suggestion
  COLLATE = utf8mb4_unicode_ci
  COMMENT = '投稿マスタ'
```